samples/multiple-inheritance/sample.md

```diff
@@ -0,0 +1,20 @@
+## Introduction to Consumer Driven Contracts
+This sample shows a [consumer driven contracts](http://martinfowler.com/articles/consumerDrivenContracts.html)(CDD) approach to messaging. The essense of consumer driven contracts is that the owner ship of the contract is inversed. Instead of the producer defining the contract consumers are now the ones defining the contract they expect and it's up to the producer to fullfill it.
+
+In the NServiceBus world message contracts is plain C# types so to honor a consumer contract the producer would just make the relevant message contract inherit from the contract type. 
```

Comment:
I chose producer/consumer since that's the lingo used in the definition of CDD. I'll add some text to explain that producer = publisher, consumer = subscriber in the NServiceBus world